arch/sim/src/sim/posix/sim_alsa.c

```diff
@@ -707,7 +701,7 @@ static void sim_audio_process(struct sim_audio_s *priv)
   else
     {
       expect = priv->playback ? apb->nbytes / priv->frame_size
-                              : AUDMIN(apb->nmaxbytes, priv->buffer_size)
+                              : MIN(apb->nmaxbytes, priv->buffer_size)
```

Comment:
Need to include `sys/param.h` in this file.

Reply:
Done.